lib/delayed/job_groups/plugin.rb

```diff
@@ -21,9 +21,7 @@ def job.max_attempts
           # If a job in the job group fails, then cancel the whole job group.
           # Need to check that the job group is present since another
           # job may have concurrently cancelled it.
-          if job.in_job_group? && job.job_group && job.job_group.failure_cancels_group?
-            job.job_group.cancel
-          end
+          job.job_group.cancel if job.in_job_group? && job.job_group && job.job_group.failure_cancels_group?
```

Comment:
Might as well use the lonely operator while we're in here.

```suggestion
          job.job_group.cancel if job.in_job_group? && job.job_group&.failure_cancels_group?
```